### PR TITLE
Review 'Tutorial topic' page

### PR DESCRIPTION
**Pull request by jdbaldry**
Update one link to avoid a redirect

Rest looks good to me.

Closes https://github.com/grafana/writers-toolkit/issues/706

### CODE REVIEW

**Review by clayton-cornell — APPROVED** [review]

File: docs/sources/structure/topic-types/tutorial/index.md [hunk 1]
```diff
@@ -114,7 +114,7 @@ Depending on the application, your tutorial's data might be:
 - In demo repositories that the user clones locally
 
 For example, the [Play with Grafana Mimir](https://grafana.com/tutorials/play-with-grafana-mimir/) tutorial provides a repository that users can clone to complete the tutorial.
-As a comparison, the [Store exemplars in Grafana Mimir](https://grafana.com/docs/mimir/latest/operators-guide/use-exemplars/storing-exemplars/) topic is a pure task that a user would follow to complete their work.
+As a comparison, the [Store exemplars in Grafana Mimir](https://grafana.com/docs/mimir/latest/manage/use-exemplars/store-exemplars/) topic is a pure task that a user would follow to complete their work.
```
Comment (josmperez) [inline docs/sources/structure/topic-types/tutorial/index.md:117]: This topic goes to great lengths to explain the distinction between a tutorial and a task-oriented topic (what is commonly referred to as a how-to guide). Then why does this sentence tell us that a tutorial can be composed purely of a task? This is confusing.

**Review by josmperez — CHANGES_REQUESTED** [review]

File: docs/sources/structure/topic-types/tutorial/index.md (hunk 1 above)
Comment (jdbaldry) [reply docs/sources/structure/topic-types/tutorial/index.md:117]: I think it is saying that _Store exemplars in Grafana Mimir_ is a task topic and not a tutorial.

Comment (josmperez) [inline docs/sources/structure/topic-types/tutorial/index.md:117]:
```suggestion
In contrast, the [Store exemplars in Grafana Mimir](https://grafana.com/docs/mimir/latest/manage/use-exemplars/store-exemplars/) topic is a pure task that a user would follow to complete their work. For guidance on writing tasks, see [Tasks](https://grafana.com/docs/writers-toolkit/structure/topic-types/task/).
```

Comment (josmperez) [reply docs/sources/structure/topic-types/tutorial/index.md:117]: In that case, it's a contrast not a comparison. I have added an appropriate suggestion.

Comment (jdbaldry) [reply docs/sources/structure/topic-types/tutorial/index.md]: Makes sense! Thanks for the suggestion, please re-review and remove the request for changes if you are happy :)

File: docs/sources/structure/topic-types/tutorial/index.md [hunk 2]
```diff
@@ -114,7 +114,7 @@ Depending on the application, your tutorial's data might be:
 - In demo repositories that the user clones locally
 
 For example, the [Play with Grafana Mimir](https://grafana.com/tutorials/play-with-grafana-mimir/) tutorial provides a repository that users can clone to complete the tutorial.
-As a comparison, the [Store exemplars in Grafana Mimir](https://grafana.com/docs/mimir/latest/operators-guide/use-exemplars/storing-exemplars/) topic is a pure task that a user would follow to complete their work.
+In contrast, the [Store exemplars in Grafana Mimir](https://grafana.com/docs/mimir/latest/manage/use-exemplars/store-exemplars/) topic is a pure task that a user would follow to complete their work. For guidance on writing tasks, see [Tasks](https://grafana.com/docs/writers-toolkit/structure/topic-types/task/).
```
[the remark this answers is not in the document]
Comment (jdbaldry) [reply docs/sources/structure/topic-types/tutorial/index.md:117]:
```suggestion
In contrast, the [Store exemplars in Grafana Mimir](https://grafana.com/docs/mimir/latest/manage/use-exemplars/store-exemplars/) topic is a pure task that a user would follow to complete their work.
For guidance on writing tasks, refer to [Tasks](https://grafana.com/docs/writers-toolkit/structure/topic-types/task/).
```

**Review by josmperez — APPROVED** [review]
LGTM.

**Pull request merged**